support-frontend/assets/components/subscriptionsProductDescription/subscriptionsProductDescription.jsx

```diff
@@ -17,6 +17,17 @@ type PropTypes = {
   buttons: ProductButton[],
 };
 
+const getAppearance = (isFeature, index, hierarchy) => {
```

Comment:
would getButtonAppearance be a better name? as this seems specific to the buttons

Reply:
Sure